.changeset/hip-kids-decide.md

```diff
@@ -0,0 +1,5 @@
+---
+"sit-onyx": major
+---
+
+feat: Remove/Rename emits that collide with native event names
```

Comment:
```suggestion
feat: Remove/Rename emits that collide with native event names

- OnyxInput, OnyxTextarea and OnyxStepper: remove `focus` and `blur` event
- OnyxInput and OnyxTextarea: `change` event is now the native HTML event so to get the input value, change `$event` to `$event.target.value`
- OnyxNavBar: rename `appAreaClick` to `navigateToStart` and `backButtonClick` to `navigateBack`
- OnyxNavButton: rename `click` to `navigate`, also the native MouseEvent is now passed as second parameter
- OnyxNavItem: rename `click` to `navigate`
- OnyxRadioButton: `change` event is now the native HTML event so to get the input value, change `$event` to `$event.target.checked`
- OnyxSelectInput: rename `click` to `inputClick`
- OnyxToastMessage: `click` event is now the native HTML event so it will always be emitted, even when `clickable` prop is not set.
```

We should be as detailed as possible for the changelog, otherwise the user will not know how to migrate to the new version

Reply:
Updated the changeset